src/parser/jobs/mnk/modules/Forms.tsx

```diff
@@ -2,13 +2,13 @@ import {Plural, Trans} from '@lingui/react'
 import React from 'react'
 
 import {ActionLink, StatusLink} from 'components/ui/DbLink'
-import {getDataBy} from 'data'
 import ACTIONS from 'data/ACTIONS'
 import STATUSES from 'data/STATUSES'
 import {BuffEvent, CastEvent} from 'fflogs'
 
 import Module, {dependency} from 'parser/core/Module'
 import Combatants from 'parser/core/modules/Combatants'
+import {Data} from 'parser/core/modules/data'
```

Comment:
Getting a compile time warning about the lower case `data` here instead of `Data` everywhere else.  Probably worth being consistent to prevent possible case-sensitive-filesystem excitement in the future.